setup.cfg

```diff
@@ -19,6 +19,6 @@ line_length = 79
 multi_line_output = 0
 known_standard_library = pkg_resources,setuptools
 known_first_party = mmaction
-known_third_party = cv2,joblib,matplotlib,mmcv,numpy,pandas,pytest,scipy,seaborn,titlecase,torch
+known_third_party = cv2,joblib,matplotlib,mmcv,moviepy,numpy,pandas,pytest,scipy,seaborn,titlecase,torch,tqdm
```

Comment:
Is it possible to remove moviepy and tqdm?

Reply:
Okay we can leave tqdm since it is small and has no further dependency.

Reply:
OK, I will remove moviepy

Reply:
Hmm mmcv has progressbar, let's use it :)

https://mmcv.readthedocs.io/en/latest/utils.html#progressbar